
El resultado es **un error**.

La funci√≥n `sayHi` se declara dentro de `if`, por lo que solo vive dentro de ella. No hay `sayHi` afuera.

